platforms/platform-alexa/docs/audioplayer.md

```diff
@@ -0,0 +1,430 @@
+---
+title: 'Alexa AudioPlayer'
+excerpt: 'Learn how to build Alexa Skills with AudioPlayer functionality using Jovo.'
+---
+
+# Alexa AudioPlayer
+
+Learn how to build Alexa Skills with AudioPlayer functionality using Jovo.
+
+## Introduction
+
+AudioPlayer Alexa Skills can be used to stream long-form audio files like music or podcasts. For this functionality, Alexa provides specific built-in intents as well as request types. More information can be found in the [official Alexa docs](https://developer.amazon.com/docs/alexa/custom-skills/audioplayer-interface-reference.html). Also, there is an [example Jovo Alexa AudioPlayer app here](https://github.com/jovotech/jovo-framework/tree/v4/latest/examples/typescript/alexa/audioplayer).
+
+In the [configuration](#configuration) section, you can learn how to set up your Alexa Skill project and Jovo app to work the Alexa AudioPlayer interface.
+
+You can interact with the AudioPlayer interface like this:
+- [Send AudioPlayer Responses](#send-audioplayer-responses): Play or stop songs and manage the queue of songs to be played.
+- [Handle AudioPlayer Requests](#handle-audioplayer-requests): Alexa sends requests to your Jovo app that contain information about the current player activity.
```

Comment:
*contains

Reply:
requests is plural

Reply:
True. Don't trust Grammarly without questioning it :D